Resources/Prototypes/_Harmony/Entities/Objects/Weapons/Bombs/plastic.yml

```
@@ -0,0 +1,31 @@
+﻿- type: entity
+  name: composition X-4
+  description: A new prototype of explosive, meant to rival a popular tool for saboteurs, C4.
+  parent: [BasePlasticExplosive, BaseSyndicateContraband]
+  id: X4
```

The order is not correct. Entity prototypes should always be defined in the order type > abstract > parent > id > categories > name > suffix > description > components.
```suggestion
  parent: [BasePlasticExplosive, BaseSyndicateContraband]
  id: X4
  name: composition X-4
  description: A new prototype of explosive, meant to rival a popular tool for saboteurs, C4.
```